snowplow/iglu-client-embedded/schemas/com.metabase/search/jsonschema/1-0-1

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Search events",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "search",
+    "format": "jsonschema",
+    "version": "1-0-1"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": [
+        "new_search_query",
+        "search_results_filtered"
+      ],
+      "maxLength": 1024
+    },
+    "runtime_milliseconds": {
+      "description": "Number of milliseconds it took to successfully run the search query",
+      "type": [
+        "integer",
+        "null"
+      ],
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "content_type": {
+      "description": "Filter for entity types (question, database, model, etc.)",
+      "type": [
+        "array",
+        "null"
+      ],
+      "items": {
+        "type": "string"
+      }
+    },
+    "creator": {
+      "description": "Filter for created by a user",
```

Comment:
@cdeweyx let me know what you'd like to see for these descriptions (if they need to be changed, anyway)

Reply:
Looks reasonable to me, I don't know of any way these are exposed in the schema itself (at least that I use in practice) so feel free to whatever makes this clearest for future engineers iterating on this. Nothing here looks wrong to me.